deploy/cluster_role.yaml

```diff
@@ -27,6 +27,7 @@ rules:
       - watch
   - apiGroups:
       - config.openshift.io
+      - console.openshift.io
     resources:
       - infrastructures
```

Comment:
Don't you add the `consolelinks` resource here ?

Reply:
I did not fail on OpenShift 3.11 but I did not investigate when this file is applied.
Will investigate and answer here why it should not fail or will rework it.

Reply:
Ah, sorry. I misunderstood your question! Good catch!

Reply:
BTW I quickly tried to test if putting extra items (which is not supported on a cluster) does not lead to errors. Seems that it does not
![Screenshot_20190902_163815](https://user-images.githubusercontent.com/5887312/64118461-66858a80-cda0-11e9-85f9-1eb4a6587977.png)
Will retest it later

Reply:
Yes, you're right afaik.